### DISCUSSION TITLE
Usage of some tokens

### DISCUSSION THREAD

**Discussion by devtrix** [skeletonlabs/skeleton#2271] [Q&A] [answered]
Can someone please explain how these design tokens such as --on-primary, --on-secondary etc are used?  I have tried to find their relevance with no luck. 
Thanks in advance.

**Comment by Sarenor** [comment, marked as answer]
We also offer a Discord if you want faster answer/a conversation.

``--on-primary`` is not a design token, it's a css variable that's used in design tokens.
So if you apply ``bg-primary-500`` to a div, the ``--on-primary`` variable should be applied to the text color.

**Reply by Sarenor** [reply]
If you wanted to know where they are actually used and defined, the tailwind plugin handles that: https://github.com/skeletonlabs/skeleton/blob/dev/packages/plugin/src/tailwind/tokens/backgrounds.ts

**Reply by devtrix** [reply]
I will note the discord channel.  I figured that may be the case but I do appreciate your patience. Thank you for your quick response.  
Happy Hollidays! Cheers
